src/pages/AddItem.js

```diff
@@ -2,26 +2,33 @@ import React, { useState, useEffect } from 'react';
 import { withFirestore } from 'react-firestore';
 import AddItemHeader from './AddItemHeader';
 import getToken from '../lib/token';
-import NavTabs from '../components/NavTabs';
```

Comment:
@mikeramz86 @MonicaDJohnson @ajiles91 Should the navigation be on each view? I didn't work on that story and it isn't clear to me if the intent is to have a persistent tab navigation or not.

Reply:
@prophen

Good question. Task 4 req was to create The  back arrow that leads back to the list view. I think it’s great to have the Nav bar in list view  but would it be redundant to have it in the additem.js too considering we are already on the additem view and we have a way to go back to the list? I’m open to hear everyone’s thoughts

Reply:
That answers my question and makes sense to me, @mikeramz86.